src/styles/partials/components/_page-header.scss

```diff
@@ -0,0 +1,67 @@
+.dg_page-header {
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: $white;
+  text-decoration: none;
+  position: relative;
+  
+  .dg_page-header-top-left {
+    position: absolute;
+    z-index: 9;
+    top: $padding-large;
+    left: $default-padding;
+  }
+  img {
+    width: 100%;
+  }
+  p {
+    color: $white;
+    font-family: $default-body-font;
+    font-size: $h2-font-size;
+    font-style: italic;
+    padding-top: 25px;
+  }
+
+  .dg_page-header__pre-title {
+    display: block;
+    font-family: $default-heading-font;
+    font-size: $smallest-font-size;
+    font-weight: $font-bold;
+    letter-spacing: $padding-small;
+    text-transform: uppercase;
+  }
+  .dg_page-header__title {
+    display: block;
+    font-family: $default-heading-font;
+    font-size: $h1-font-size;
+    font-weight: $font-bold;
+    letter-spacing: $base-spacing-unit;
+    text-transform: uppercase;
+  }
+  &:after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: none;
+    color: $white;
+    display: block;
+    background: transparent;
+  }
+
+  &.blue:after {
+    background: rgba($blue, 0.7);
+  }
+
+  &.red:after {
```

Comment:
this was jsut an example, we actually don't need red at this time, you can remove.